index.html

```diff
@@ -13,196 +13,217 @@
 <body>
     <header>
         <a class="logo" href="#">My Logo</a>
-        <nav class="menu"><img src="images/Union.png" alt="menu"></nav>
+        <nav class="menu"><a href="#"><img src="images/Union.png" alt="menu"></a></nav>
+        <nav class="menu1">
+            <ul>
+                <li><a href="#portfolio">Portfolio</a> </li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Kindly, change some `padding-top` and `font-size` for our menus to have the same we have on the figma.

----------------------------------------------Yours----------------------------------------------------------
![port31](https://user-images.githubusercontent.com/55635977/130134204-f7bf2be7-9caa-413d-ba03-394532248427.png)
----------------------------------------------Original----------------------------------------------------------
![port33](https://user-images.githubusercontent.com/55635977/130135019-0cceefd9-1df4-42c9-8e3a-9f96c137f254.png)

Reply:
You reviewing me unfair. why can't you understand that the instructions of this project was to try make is the same as possible

Reply:
I know you can make it, you only have to have to change the font-size and padding-top value :pray:

Reply:
I already did that.
Thanks alot for the review